app/src/main/java/com/connfa/model/database/AbstractEntityDAO.java

```diff
@@ -25,31 +32,29 @@
     /**
      * This method should return list of columns, used to define unique object
      * in "contains" method
-     *
-     * @return
      */
     protected abstract String[] getKeyColumns();
 
-    protected ILAPIDBFacade getFacade() {
-        ILAPIDBFacade dbFacade = LAPIDBRegister.getInstance().lookup(getDatabaseName());
+    protected Context getContext() {
```

Comment:
Make it final plz (this and all the methods that aren't overridden in subclasses)